pydicom/sr/derived_value_types.py

```diff
@@ -0,0 +1,331 @@
+            name=CodedConcept(
+                value='121232',
+                meaning='Source Series for Segmentation',
+                scheme_designator='DCM'
+            ),
+            value=series_instance_uid,
+            relationship_type=RelationshipTypes.CONTAINS
+        )
+
+
+class ReferencedRegionContentItem(ScoordContentItem):
+
+    """Content item for a refrenced region of interest"""
+
+    def __init__(self, graphic_type, graphic_data, source_image,
+                 pixel_origin_interpretation=PixelOriginInterpretations.VOLUME):
+        """
+        Parameters
+        ----------
+        graphic_type: Union[pydicom.sr.value_types.GraphicTypes, str]
+            name of the graphic type
+        graphic_data: List[List[int]]
+            ordered set of (row, column) coordinate pairs
+        source_image: pydicom.sr.template.SourceImageForRegionContentItem
+            source image to which `graphic_data` relate
+        pixel_origin_interpretation: Union[pydicom.sr.value_types.PixelOriginInterpretations, str, None], optional
+            whether pixel coordinates specified by `graphic_data` are defined
+            relative to the total pixel matrix
+            (``pydicom.sr.value_types.PixelOriginInterpretations.VOLUME``) or
+            relative to an individual frame
+            (``pydicom.sr.value_types.PixelOriginInterpretations.FRAME``)
+            of the source image
+            (default: ``pydicom.sr.value_types.PixelOriginInterpretations.VOLUME``)
+
+        """  # noqa
+        graphic_type = GraphicTypes(graphic_type)
+        if graphic_type == GraphicTypes.MULTIPOINT:
+            raise ValueError(
+                'Graphic type "MULTIPOINT" is not valid for region.'
+            )
+        if not isinstance(source_image, SourceImageForRegionContentItem):
+            raise TypeError(
+                'Arguement "source_image" must have type '
+                'SourceImageForRegionContentItem.'
+            )
+        if pixel_origin_interpretation == PixelOriginInterpretations.FRAME:
+            if (not hasattr(source_image, 'ReferencedFrameNumber') or
+                source_image.ReferencedFrameNumber is None):
+                raise ValueError(
+                    'Frame number of source image must be specified when value '
+                    'of argument "pixel_origin_interpretation" is "FRAME".'
+                )
+        super(ReferencedRegionContentItem, self).__init__(
+            name=CodedConcept(
+                value='111030',
+                meaning='Image Region',
+                scheme_designator='DCM'
+            ),
+            graphic_type=graphic_type,
+            graphic_data=graphic_data,
+            pixel_origin_interpretation=pixel_origin_interpretation,
+            relationship_type=RelationshipTypes.CONTAINS
+        )
+        self.ContentSequence = [source_image]
+
+
+class ReferencedVolumeContentItem(Scoord3DContentItem):
```

Comment:
Is it correct that `ReferencedVolumeContentItem` is inherited from `Scoord3DContentItem`?